diff -ur calamares-3.2.8/src/modules/bootloader/main.py calamares-3.2.8-shim-grub-cfg/src/modules/bootloader/main.py
--- calamares-3.2.8/src/modules/bootloader/main.py	2019-05-11 01:00:11.000000000 +0200
+++ calamares-3.2.8-shim-grub-cfg/src/modules/bootloader/main.py	2019-05-12 15:26:16.944784471 +0200
@@ -7,7 +7,7 @@
 #   Copyright 2014, Anke Boersma <demm@kaosx.us>
 #   Copyright 2014, Daniel Hillenbrand <codeworkx@bbqlinux.org>
 #   Copyright 2014, Benjamin Vaudour <benjamin.vaudour@yahoo.fr>
-#   Copyright 2014, Kevin Kofler <kevin.kofler@chello.at>
+#   Copyright 2014-2019, Kevin Kofler <kevin.kofler@chello.at>
 #   Copyright 2015-2018, Philip Mueller <philm@manjaro.org>
 #   Copyright 2016-2017, Teo Mrnjavac <teo@kde.org>
 #   Copyright 2017, Alf Gaida <agaida@siduction.org>
@@ -339,8 +339,8 @@
                                "--force",
                                boot_loader["installPath"]])
 
-    # The file specified in grubCfg should already be filled out
-    # by the grubcfg job module.
+    # The input file /etc/default/grub should already be filled out by the
+    # grubcfg job module.
     check_target_env_call([libcalamares.job.configuration["grubMkconfig"],
                            "-o", libcalamares.job.configuration["grubCfg"]])
 
@@ -395,6 +395,12 @@
         "-p", efi_partition_number,
         "-l", install_efi_directory + "/" + install_efi_bin])
 
+    # The input file /etc/default/grub should already be filled out by the
+    # grubcfg job module.
+    check_target_env_call([libcalamares.job.configuration["grubMkconfig"],
+                           "-o", os.path.join(efi_directory, "EFI",
+                                              efi_bootloader_id, "grub.cfg")])
+
 
 def vfat_correct_case(parent, name):
     for candidate in os.listdir(parent):
